src/controller/DevicePairingDelegate.h

```diff
@@ -137,6 +137,16 @@ class DLL_EXPORT DevicePairingDelegate
      * @param[in] icdCounter   The ICD Counter received from the device.
      */
     virtual void OnICDRegistrationComplete(NodeId icdNodeId, uint32_t icdCounter) {}
+
+    /**
+     * @bried
+     *   Called when the stay active response is received in the end of the LIT ICD commissioning flow completes.
```

Comment:
This needs to be clear about the fact that this might not get called because there might not be a StayActiveRequest to start with.

Reply:
agree, update the comments, thanks